third_party/xla/xla/stream_executor/rocm/rocm_driver.cc

```diff
@@ -895,7 +895,7 @@ static hipMemAllocationType ToHipAllocationType(
 
 /*static*/ absl::Status GpuDriver::GraphAddMemFreeNode(
     GpuGraphNodeHandle* node, GpuGraphHandle graph,
-    absl::Span<GpuGraphNodeHandle> deps, GpuDevicePtr gpu_dst) {
+    absl::Span<const GpuGraphNodeHandle> deps, GpuDevicePtr gpu_dst) {
```

Comment:
shall we need to upstream these changes on rocm_driver.cc as well? or is already in upstream's one of pending PRs?

Reply:
https://github.com/openxla/xla/commit/502db1d7c3f14b69f45354190f89876575dabefa